solutions/USACO feb19-herding-bronze.mdx

```diff
@@ -0,0 +1,45 @@
+---
+id: usaco-915
+source: USACO Bronze 2019 February
+title: Sleepy Cow Herding
+author: Ananth Kashyap
+---
+
+[Official Analysis](http://www.usaco.org/current/data/sol_herding_bronze_feb19.html)
+
+## Implementation
+
+**Time Complexity:** $\mathcal{O}(1)$
+
+<LanguageSection>
+<PySection>
+
+```python
+# Created by Ananth Kashyap
+
+# Take in input using python file i/o system
+file_in = open('herding.in')
+data = file_in.read().strip().split('\n')
+cows = list(map(int, data[0].split(' ')))
+
+# Determine the min and max number of moves FJ needs to make
+if cows[2] - cows[0] == 2:
+	min = 0
```

Comment:
it might not be the best idea to name this `min` and `max`- maybe `min_moves` or `max_moves`, or just `min_` and `max_` if you really want to be concise